compiler/main.nim

```diff
@@ -61,6 +61,10 @@ proc commandCheck(graph: ModuleGraph) =
   compileProject(graph)
 
   if conf.symbolFiles != disabledSf:
+    if not conf.m.trackPosProvided:
+      globalError(graph.config, unknownLineInfo,
+                  "Position required use --track or --trackDirty")
```

Comment:
We can instead change ``nim check --ic:on --track:X --defusage`` to ``nim check --ic:on defusage:X``. That makes more sense anyway.

Reply:
done.